manifests/init.pp

```diff
@@ -80,7 +80,7 @@
   $epel_gpg_managed                       = $epel::params::epel_gpg_managed,
   $os_maj_release                         = $epel::params::os_maj_release,
 ) inherits epel::params {
-  if "${::osfamily}" == 'RedHat' and "${::operatingsystem}" !~ /Fedora|Amazon/ { # lint:ignore:only_variable_string
```

Comment:
The module now explodes if you try to use it on Fedora instead of hitting the `notice ("Your operating system ${::operatingsystem} will not have the EPEL repository applied")` line.

This is causing build failures in `puppet-letsencrypt`.